### DISCUSSION TITLE
Custom Client Upgrade

### DISCUSSION THREAD

**Discussion by mahescho** [rustdesk/rustdesk-server-pro#366] [Q&A]
Is there any way to auto upgrade custom clients? If not what is the recommended way to do custom client upgrades?

**Comment by rustdesk** [comment]
Auto upgrade of custom client is almost infeasible, AnyDesk etc can not provide this either.

So far I do not have suggestions either.

**Comment by risaw1981** [comment]
> Is there any way to auto upgrade custom clients? If not what is the recommended way to do custom client upgrades?

Re-generate the installer in web console and install over the top of the client, no need to uninstall first
